src/backend/distributed/deparser/deparse_role_stmts.c

```diff
@@ -237,7 +237,7 @@ AppendCreateRoleStmt(StringInfo buf, CreateRoleStmt *stmt)
 
 		if (strcmp(option->defname, "sysid") == 0)
 		{
-			appendStringInfo(buf, " SYSID %s", quote_literal_cstr(strVal(option->arg)));
+			/* pg support that option only for backward compatibility, but actually ignore it */
```

Comment:
what is the problem with deparsing `SYSID`? Wouldn't workers ignore anyway?

Reply:
PG documentation tells that it ignores the option even if it accepts it. And when I check the code, they donot even parse the option's value, instead hey log the below notice message if it is given. 
```sql
create role xxx sysid 123;
NOTICE:  SYSID can no longer be specified
CREATE ROLE
```

Reply:
We had segfault because we are trying to parse the option value as string but it is int. Maybe we can still pass the sysid after parsing it as int.